react/src/App.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react';
+
+import { Endpoints } from './Endpoints';
+
+import { ConnectButton, Config } from 'react-pelm-connect';
+
+type State = {
+    isLoading: boolean;
+    error?: string;
+    connectToken?: string;
+    accessed: boolean;
+};
+
+export class App extends React.Component<{}, State> {
+    constructor(props = {}) {
+        super(props);
+
+        this.state = {
+            isLoading: true,
+            error: undefined,
+            connectToken: undefined,
+            accessed: false,
+        };
+    }
+
+    componentDidMount(): void {
+        this.generateConnectToken();
+    }
+
+    // Create a connect token using client credentials in order to initiate the Connect Flow.
+    generateConnectToken() {
+        this.setState({ isLoading: true });
+
+        fetch('/connect')
+            .then((r) => r.json())
+            .then((d) => {
+                this.setState({ isLoading: false });
+                this.setState({ connectToken: d.connectToken });
+            });
+    }
+
+    // Use the authorizationCode to get an access_token and refresh_token.
```

Comment:
`refresh_token` is deprecated. Let's remove that

Reply:
removed